docs/docs/cmd/purview/retentioneventtype/retentioneventtype-list.md

```diff
@@ -0,0 +1,72 @@
+# purview retentioneventtype list
+
+Get a list of retention event types
+
+## Usage
+
+```sh
+m365 purview retentioneventtype list [options]
+```
+
+## Options
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Get a list of retention event types
+
+```sh
+m365 purview retentioneventtype list
+```
+
+## Remarks
+
+!!! attention
+    This command is based on a Microsoft Graph API that is currently in preview and is subject to change once the API reached general availability.
+!!! attention
+    This command is part of a series of commands that have to do with event-based retention. Event-based retention is about starting a retention period when a specific event occurs, instead of the moment a document was labeled or created.
+
+## Response
+
+
+=== "JSON"
+
+    ```json
+    [
+      {
+        "displayName": "Retention Event Type",
```

Comment:
Let's use 'Contract Expiry Event' for dummy data here. That speaks a bit more as to what this is all about.